src/test/java/com/search/DepthFirstSearchTest.java

```diff
@@ -0,0 +1,36 @@
+package src.test.java.com.search;
+
+import org.junit.Assert;
+import org.junit.Test;
+import src.main.java.com.search.DepthFirstSearch;
+import src.main.java.com.search.BinaryTree;
```

Comment:
'src.main.java.com.search.BinaryTree' is not public in 'src.main.java.com.search'. Cannot be accessed from outside package.